ledger/simulation/tracer.go

```diff
@@ -140,6 +140,12 @@ func (tracer *evalTracer) BeforeTxnGroup(ep *logic.EvalParams) {
 		tracer.result.TxnGroups[0].AppBudgetAdded = uint64(*ep.PooledApplicationBudget)
 	}
 
+	// Override transaction group budget if specified in request, retreve from tracer.result
```

Comment:
nit: spelling
```suggestion
	// Override transaction group budget if specified in request, retrieve from tracer.result
```

Reply:
done here [renaming from allow-* to extra-app-budget](https://github.com/algorand/go-algorand/pull/5354/commits/fb69bc4cd999c09b4aa70c41e5d6a4083376502f)